events/schema.py

```diff
@@ -524,6 +527,97 @@ class EventTicketSystemInput(graphene.InputObjectType):
     )
 
 
+class ImportTicketSystemPasswordErrorType(graphene.Scalar):
+    @staticmethod
+    def serialize(errors):
+        if isinstance(errors, dict):
+            if errors.get("__all__", False):
+                errors["non_field_errors"] = errors.pop("__all__")
+            return camelize(errors)
+        raise Exception("`errors` should be dict!")
```

Comment:
Maybe `TypeError` could be a bit more appropriate?

Reply:
yep, refactored this